doc/how.rst

```diff
@@ -133,6 +134,10 @@ This ``stop_strategy`` can be:
       the running time. The parameter is called ``tol`` and should be
       a positive float.
 
+    - ``'callback'``: in this case, the ``run`` method of the solver
+      will call at each iteration a callback function that computes the
+      objective until a high number of iteration is reached.
```

Comment:
```suggestion
    - ``'callback'``: in this case, the ``run`` method of the solver
      should call at each iteration the provided callback function. It will
      compute and store the objective and return False once the computations
      should stop.
```